modules/server/templates/configs/forseti_conf_server.yaml.tpl

```diff
@@ -374,8 +376,28 @@ notifier:
                 # gcs_path should begin with "gs://"
                 gcs_path: gs://${FORSETI_BUCKET}/scanner_violations
 
-        - resource: groups_violations
+        - resource: kms_violations
           should_notify: true
+          notifiers: ${KMS_VIOLATIONS_SHOULD_NOTIFY}
+            # Email violations
```

Comment:
It looks like there's a two space indentation here which will cause this to be invalid YAML

Reply:
Resolved- thanks Adrien.